src/notifications/datamodel/models.py

```diff
@@ -29,6 +35,15 @@ class Meta:
     def __str__(self) -> str:
         return f"{self.naam}"
 
+    def match_filter_names(self, obj_filters):
```

Comment:
Type hints make the intent of code a lot clearer:
```suggestion
    def match_filter_names(self, obj_filters: list) -> bool:
```

Reply:
fixed